src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -5960,7 +6124,7 @@ def test_vmss_disk_encryption_e2e(self, resource_group, resource_group_location,
         self.kwargs.update({
             'vmss': 'vmss1'
         })
-        self.cmd('vmss create -g {rg} -n {vmss} --image Win2022Datacenter --instance-count 1 --admin-username clitester1 --admin-password Test123456789! --orchestration-mode Uniform')
+        self.cmd('vmss create -g {rg} -n {vmss} --image Win2022Datacenter --instance-count 1 --admin-username clitester1 --admin-password Test123456789! --orchestration-mode Uniform --lb-sku Standard')
```

Comment:
> --lb-sku Standard

Why do we need to add it?

Reply:
There are several ways to disable outbound access. For VMSS, one way is to use Standard load balancer.

Reply:
Got it, thanks!